specification/appconfiguration/resource-manager/readme.md

```diff
@@ -115,7 +115,6 @@ swagger-to-sdk:
   - repo: azure-sdk-for-python-track2
   - repo: azure-sdk-for-net-track2
   - repo: azure-sdk-for-java
-  - repo: azure-sdk-for-net
```

Comment:
There is a track 2 mgmt SDK, though: Azure.ResourceManager.AppConfiguration

Reply:
azure-sdk-for-net-track2 was configured before at line 116.